client/dashboard/sites/settings-wp-version/utils.ts

```diff
@@ -0,0 +1,5 @@
+import { Site } from '../../data/types';
+
+export function canUpdateWordPressVersion( site: Site ) {
```

Comment:
I have an idea to create a new endpoint to return the features that are supported by the specific site. With this endpoint, we don't need to add this kind of function on frontend. What do you think?

Reply:
Ohh, good idea! Maybe it could be a field to the existing get site endpoint instead? Like 

```js
"hosting_features": [
  "update_wp_version",
}
```

Reply:
Ohh, that also makes sense and the behavior would be the same if we simply passed `fields=hosting_features` to the existing get site endpoint!

Reply:
Yes, that's what I mean!

Reply:
But if we want to add the new field to the get site endpoint, we might need to implement it in Jetpack. The tricky part is that some features aren’t available through Jetpack 🥲

Reply:
Ohh, I see what you mean.... Good luck then please see what will work 🙈

Reply:
The reason I suggested it to be a field of `Site`, is because it's currently very nice to be able to check for features given only a `site`, like `site.is_wpcom_atomic` like we currently are doing.